cristin-project/src/test/java/no/unit/nva/cristin/projects/project.feature

```diff
@@ -0,0 +1,51 @@
+Feature: API tests for Cristin Project retrieve and search
```

Comment:
There is api tests for projects in nva-api-testing in branch update-dto-description. Maybe we can add them here later